charts/mod-data-export/values.yaml

```diff
@@ -80,6 +80,44 @@ jmx:
 
 heapDumpEnabled: false
 
+volumeClaims:
+  - enabled: false
+    name: extendedTmp
+    storageClassName: gp2
+    size: 20Gi
+    accessModes: ReadWriteOnce
+    existingClaim:
+
+extraVolumeMounts:
+  - enabled: false
+    name: heapdump
+    mountPath: /tmp/dump
+  - enabled: false
+    name: extendedTmp
+    mountPath: /tmp
+
+extraVolumes:
+  - enabled: false
+    name: heapdump
+    emptyDir: {}
+  - enabled: false
+    name: extendedTmp
+    persistentVolumeClaim:
+      claimName: extendedTmp
+
+initContainer:
```

Comment:
@epam-avramenko 
Is there any reason we have to use **InitContainer** run insted of **spec.securityContext.fsGroup** here ?
https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
Running extra container takes more time and CPU resources.

Example for POD:
spec:
  securityContext:
    runAsUser: 1000
    runAsGroup: 3000
    fsGroup: 2000

Reply:
I understand your concern about resource consumption and the different approaches we could potentially introduce for this solution. In my humble opinion, it would be better to use an init container instead of a security context to avoid potential security breaches that may occur when granting the container root permissions. The init container won't impact resource consumption, as it will be terminated after its initial performance. The only drawback I see is the additional time needed for its execution. On the other hand, the init container feature, which was introduced in the https://github.com/folio-org/folio-helm-v2/pull/115 pull request, could be used in other potential cases. Additionally, leveraging the init container is a common approach in similar cases according to different sources and different Bitnami chart implementations. While the security context is an option, let's stick with the init container approach.

Reply:
Thank you.
I've got your point.
Let it work for a while untill we make sure approach with **security.Context.fsGroup** works for us :)